about/system_requirements.rst

```diff
@@ -0,0 +1,116 @@
+|                      |                                                                                         |
+|                      | - **Forward Mobile rendering method:** Integrated graphics with full Vulkan 1.0 support |
+|                      |                                                                                         |
+|                      |   - *Example: Intel GD Graphics 5500 (Broadwell), AMD Radeon R5 Graphics (Kaveri)*      |
+|                      |                                                                                         |
+|                      | - **Compatibility rendering method:** Integrated graphics with full OpenGL 3.3 support  |
+|                      |                                                                                         |
+|                      |   - *Example: Intel GD Graphics 2500 (Ivy Bridge), AMD Radeon R5 Graphics (Kaveri)*     |
++----------------------+-----------------------------------------------------------------------------------------+
+| **RAM**              | - **For native exports:** 2 GB                                                          |
+|                      | - **For web exports:** 4 GB                                                             |
++----------------------+-----------------------------------------------------------------------------------------+
+| **Storage**          | 100 MB (used for the executable, project files and cache)                               |
++----------------------+-----------------------------------------------------------------------------------------+
+| **Operating system** | - **For native exports:** Windows 7, or Linux distribution released after 2016          |
+|                      | - **For web exports:** Firefox 79, Chrome 68, Edge 79, Safari 15.2, Opera 64            |
++----------------------+-----------------------------------------------------------------------------------------+
+
+Mobile device (smartphone/tablet)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++----------------------+-----------------------------------------------------------------------------------------+
+| **CPU**              | - **Android:** SoC with any 32-bit or 64-bit ARM or x86 CPU                             |
+|                      |                                                                                         |
+|                      |   - *Example: Qualcomm Snapdragon 430, Samsung Exynos 5 Octa 5430*                      |
+|                      |                                                                                         |
+|                      | - **iOS:** SoC with any 64-bit ARM CPU                                                  |
+|                      |                                                                                         |
+|                      |   - *Example: Apple A7 (iPhone 5S)*                                                     |
```

Comment:
For Vulkan renderer it's likely A9+ (iPhone 6S / SE 1.gen.), A7 have Metal support, but not all features used by Godot are supported.